tests/test_ax_generators.py

```diff
@@ -199,6 +199,75 @@ def test_ax_single_fidelity():
     make_plots(gen)
 
 
+def test_ax_single_fidelity_resume():
```

Comment:
It seems that this function was simply moved from its previous location in this file to this new location.

If this is indeed the case, could you move it back to its previous location?